scripts/create-symlinks.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Can we get away with `#!/bin/sh`?

Reply:
No, there are bashisms like the files array below.

Reply:
@fchavant this works for me with `sh`.

Reply:
@alepop Your default shell is probably bash (or at least bash compatible). If you run `sh --version` does it tell you it's bash?